### ISSUE TITLE
Failure to build on High Sierra

### ISSUE THREAD

**Issue by ViralBShah** [JuliaInterop/ZMQ.jl#157]
I have `pkg-config` installed, but that doesn't seem to register here. 

```
julia> Pkg.build("IJulia")
INFO: Building Conda
INFO: Building Homebrew
INFO: Building MbedTLS
INFO: Building ZMQ
WARNING: Compat.KERNEL is deprecated.
  likely near /Users/viral/.julia/v0.6/ZMQ/deps/build.jl:36
WARNING: Compat.KERNEL is deprecated.
  likely near /Users/viral/.julia/v0.6/ZMQ/deps/build.jl:36
in can_use at /Users/viral/.julia/v0.6/Homebrew/src/bindeps_integration.jl
WARNING: Compat.KERNEL is deprecated.
  likely near /Users/viral/.julia/v0.6/ZMQ/deps/build.jl:36
in package_available at /Users/viral/.julia/v0.6/Homebrew/src/bindeps_integration.jl
WARNING: The following packages do not have relocatable bottles, installation may fail!
Please report these packages to https://github.com/JuliaLang/Homebrew.jl:
  staticfloat/juliadeps/zeromq32
Error: No such file or directory - pkg-config
=================================[ ERROR: ZMQ ]=================================

LoadError: failed process: Process(`/Users/viral/.julia/v0.6/Homebrew/deps/usr/bin/brew install --ignore-dependencies staticfloat/juliadeps/zeromq32`, ProcessExited(1)) [1]
while loading /Users/viral/.julia/v0.6/ZMQ/deps/build.jl, in expression starting on line 36

================================================================================
INFO: Building IJulia
INFO: Installing Jupyter via the Conda package.
INFO: Found Jupyter version 4.3.0: /Users/viral/.julia/v0.6/Conda/deps/usr/bin/jupyter
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 12309  100 12309    0     0  79255      0 --:--:-- --:--:-- --:--:-- 79412
INFO: Installing Julia kernelspec julia-0.6
[InstallKernelSpec] Removing existing kernelspec in /Users/viral/Library/Jupyter/kernels/julia-0.6
[InstallKernelSpec] Installed kernelspec julia-0.6 in /Users/viral/Library/Jupyter/kernels/julia-0.6

================================[ BUILD ERRORS ]================================

WARNING: ZMQ had build errors.

 - packages with build errors remain installed in /Users/viral/.julia/v0.6
 - build the package(s) and all dependencies with `Pkg.build("ZMQ")`
 - build a single package by running its `deps/build.jl` script

================================================================================

```

**Comment by ghuba** [comment]
Same problem for me. Build fail with similar error message:

```
Error: No such file or directory - pkg-config
=======================================================================================================================================================[ ERROR: ZMQ ]========================================================================================================================================================

LoadError: failed process: Process(`/Users/.../.julia/v0.6/Homebrew/deps/usr/bin/brew install --ignore-dependencies staticfloat/juliadeps/zeromq32`, ProcessExited(1)) [1]
while loading /Users/.../.julia/v0.6/ZMQ/deps/build.jl, in expression starting on line 36
```

**Comment by ghuba** [comment]
May be related to

https://github.com/staticfloat/homebrew-julia/issues/240

**Comment by plumer** [comment]
Same problem here. I attempted to install IJulia and eventually boiling down to this ZMQ issue.

**Comment by ghuba** [comment]
Can you check this? It fixed the issue for me.

https://github.com/JuliaPackaging/Homebrew.jl/issues/202

`using Homebrew; Homebrew.add("zeromq@3.2")`

**Comment by ViralBShah** [comment]
Fixed in #158. Needs a new tag.

**Comment by aviks** [comment]
Fixed by `v0.5.1`
